app/views/students/_student_form.html.haml

```diff
@@ -1,10 +1,38 @@
 %tr
-  %td.pt-3
-    %h5= form_type.to_s.titleize
-  - if @student.valid_forms(form_type).any?
-    %td.text-center.pt-3.table-success
-      %h5= "Valid"
+  - if form_type != :stims
+    %td.pb-1.pt-3
+      %p.lead= form_type.to_s.titleize
+    - if @student.valid_forms(form_type).any?
+      %td.text-center.pb-1.pt-3.table-success
+        %p.lead= "Valid"
+    - else
+      %td.text-center.pb-1.pt-3.table-danger
+        %p.lead.font-weight-bold= "Missing"
   - else
-    %td.text-center.pt-3.table-danger
-      %h5
-        %strong= "Missing"
+    %td.pb-1.pt-3
+      %p.lead On Team
+    - stims = @student.valid_forms(:stims)
```

Comment:
The stim returned here can be reused. `valid_forms` returns an [`ActiveRecord::Relation`](http://api.rubyonrails.org/v5.0/classes/ActiveRecord/Relation.html), which behaves like an array.

So you can pull the stim:
```
stim = @student.valid_forms(:stims).take # assumes we only ever get one stim back
```
and then your checks below don't need scopes (and therefore don't need to re-query the database).

```
if stim.has_guardian ...
if stim.consented ...
```